__tests__/__fixtures__/output/javascript/axios/application-form-encoded.js

```diff
@@ -1,10 +1,14 @@
 import axios from "axios";
 
+const encodedParams = new URLSearchParams();
+encodedParams.set('foo', 'bar');
+encodedParams.set('hello', 'world');
+
 const options = {
   method: 'POST',
   url: 'https://httpbin.org/anything',
   headers: {'content-type': 'application/x-www-form-urlencoded'},
-  data: {foo: 'bar', hello: 'world'}
```

Comment:
We didn't know that this was wrong because we only test Axios on Node in CI, not the JS version because it uses an `import` syntax. 😰

Reply:
Huh so axios does different things depending on whether it's being run in Node or the Browser?

Reply:
No this brings the JS Axios example up to pairity with the Node one that was already using UrlSearchParams